usr/bin/playtron-weston

```diff
@@ -3,4 +3,10 @@
 # the dev session switch is a one time thing, immediately
 # revert to the default/user session for next boot
 pkexec playtronos-session-select user --no-switch
-weston
+
+SYS_ID="$(cat /sys/devices/virtual/dmi/id/product_name)"
+if [[ ":AYANEO 2S:SUIPLAY 0X1:" =~ ":$SYS_ID:"  ]]; then
```

Comment:
I wanted to capture this before we forget: the current product name is "SuiPlay0X1" (where the 0 is a zero and not an oh).

Reply:
Updated.